src/lib/middleware/framework-services/restify.ts

```diff
@@ -31,10 +31,15 @@ export default class RestifyService implements FrameworkService {
     }
 
     getResHeaderField(res: any, fieldName: string): string {
-        return res.get ? res.get(fieldName) : "";
+        return res.get ? res.get(fieldName) : undefined;
```

Comment:
I have updated the return type. We introduced the checks quite some time ago to prevent errors when an incorrect framework is selected. I have added another commit to align the implementations for the different frameworks.